rename everything to not use `calc`

hmacdope opened the pull request:
Rename calc_bonds to calc_distances, more correct name

@richardjgowers should we keep the Calc prefix? Now is the time to remove even if labour intensive

pep8speaks commented:
Hello @hmacdope! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:



* In the file [`distopia/tests/test_distopia.py`](https://github.com/MDAnalysis/distopia/blob/c96e7a3f0333f74c56c56e58f3426eb445c5a948/distopia/tests/test_distopia.py):

> [Line 33:80](https://github.com/MDAnalysis/distopia/blob/c96e7a3f0333f74c56c56e58f3426eb445c5a948/distopia/tests/test_distopia.py#L33): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (83 > 79 characters)
> [Line 61:80](https://github.com/MDAnalysis/distopia/blob/c96e7a3f0333f74c56c56e58f3426eb445c5a948/distopia/tests/test_distopia.py#L61): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (82 > 79 characters)
> [Line 66:80](https://github.com/MDAnalysis/distopia/blob/c96e7a3f0333f74c56c56e58f3426eb445c5a948/distopia/tests/test_distopia.py#L66): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (86 > 79 characters)
> [Line 86:80](https://github.com/MDAnalysis/distopia/blob/c96e7a3f0333f74c56c56e58f3426eb445c5a948/distopia/tests/test_distopia.py#L86): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (89 > 79 characters)
> [Line 95:80](https://github.com/MDAnalysis/distopia/blob/c96e7a3f0333f74c56c56e58f3426eb445c5a948/distopia/tests/test_distopia.py#L95): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (93 > 79 characters)
> [Line 104:80](https://github.com/MDAnalysis/distopia/blob/c96e7a3f0333f74c56c56e58f3426eb445c5a948/distopia/tests/test_distopia.py#L104): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (97 > 79 characters)
> [Line 375:80](https://github.com/MDAnalysis/distopia/blob/c96e7a3f0333f74c56c56e58f3426eb445c5a948/distopia/tests/test_distopia.py#L375): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (87 > 79 characters)
> [Line 378:80](https://github.com/MDAnalysis/distopia/blob/c96e7a3f0333f74c56c56e58f3426eb445c5a948/distopia/tests/test_distopia.py#L378): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (96 > 79 characters)
> [Line 380:80](https://github.com/MDAnalysis/distopia/blob/c96e7a3f0333f74c56c56e58f3426eb445c5a948/distopia/tests/test_distopia.py#L380): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (100 > 79 characters)
> [Line 397:80](https://github.com/MDAnalysis/distopia/blob/c96e7a3f0333f74c56c56e58f3426eb445c5a948/distopia/tests/test_distopia.py#L397): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (108 > 79 characters)

richardjgowers commented:
@hmacdope yeah I'm not wildly in love with calc_bonds as a name, but calc_distances is perhaps more vague, most the functions are calculating a distance of some sort.  How about zip_distances? the logic being it's a bit like `distance(a,b) for a, b in zip(c1, c2)`

richardjgowers commented:
also yeah `calc_` everywhere is tiresome

hmacdope commented:
@richardjgowers I have removed `Calc` from everywhere, but I haven't gone through and added `Zip` to the relevant ones. Would it be so bad to go ahead without it?

richardjgowers commented:
@hmacdope Sounds good!
